demisto_sdk/common/hook_validations/release_notes.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import print_function
+
+import os
+import re
+from demisto_sdk.common.tools import run_command
+
+from demisto_sdk.common.tools import print_error, get_latest_release_notes_text, \
+    get_release_notes_file_path
+
+release_notes_standard = 'https://github.com/demisto/content/blob/master/docs/release_notes/README.md'
+
+
+class ReleaseNotesValidator:
+    """Release notes validator is designed to ensure the existence and correctness of the release notes in content repo.
+
+    Attributes:
+        file_path (str): the path to the file we are examining at the moment.
+
+    """
+
+    def __init__(self, file_path):
+        # type: (str) -> None
+        self.file_path = file_path
+        self.release_notes_path = get_release_notes_file_path(self.file_path)
+        self.release_notes = get_latest_release_notes_text(self.release_notes_path)
+        self.master_diff = self.get_master_diff()
+
+    def get_master_diff(self):
+        # type: () -> str
+        """Gets difference between current branch and origin/master
+
+        git diff with the --unified=10000 option means that if there exists a
+        difference between origin/master and current branch, the output will have at most 10000
+        lines of context.
```

Comment:
why do we want so many? i think that 20 should do

Reply:
it doesn't matter only if we take into consideration that the only part of the changelog file that will be changed is the `Unreleased` part. Otherwise I believe that 20 is insufficient. 10000 is just a matter of safety.

Reply:
i don't want us to reach a point in which that function will give the entire changelog for some reason and then it will read 10000 which will take forever, lets put a more secure limit here, 100 or something of that sort, it should not be longer then even 10